tests/integration-service/README.md

```diff
@@ -0,0 +1,154 @@
+# Integration Service
+
+The Integration Service manages the lifecycle, deployment and integration of applications and components. It handles snapshots, environments, pipelines and deployments, ensuring applications run efficiently and are deployed correctly.
+
+
+## Happy Path Tests.
```

Comment:
Can we add a quick sentence under here saying what happy path tests are?  Unless its defined in a higher-level README in this repo.